packages/renderer/src/lib/pvc/PVCColumnMode.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+import { faCircle, faEye, faLock, faSitemap, faTh } from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa';
+
+import Label from '../ui/Label.svelte';
+import type { PVCUI } from './PVCUI';
+
+export let object: PVCUI;
+
+// Determine the icon and color based on the access mode, with comments explaining each icon choice
+function getModeAttributes(mode: string) {
+  switch (mode) {
+    case 'ReadWriteOnce':
+      // faLock: Represents exclusive write access (one writer), indicative of the ReadWriteOnce mode
+      return { color: 'text-green-600', icon: faLock };
```

Comment:
These should really be pulling from the color registry now. I guess we can defer, but maybe open a follow-up issue to create color vars for all of the Kube objects?